common/lib/xmodule/xmodule/modulestore/tests/utils.py

```diff
@@ -197,3 +198,40 @@ def assert_xss(self, response, xss_content):
         """
         self.assertContains(response, escape(xss_content))
         self.assertNotContains(response, xss_content)
+
+
+class MockSignalHandlerMixin(object):
```

Comment:
I really like the idea that we have a Mixin that will handle signals, but since we're using django signals, I'm not sure if makes sense to put this in `common/lib/xmodule`.